<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Encode</class>
 <widget class="QDialog" name="Encode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert data to file</string>
  </property>
  <widget class="QLineEdit" name="containerFileLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Stego.bmp</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataFileLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>data.txt</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>42</string>
   </property>
  </widget>
  <widget class="QLabel" name="containerLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Container File</string>
   </property>
  </widget>
  <widget class="QLabel" name="dataFileLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Data File</string>
   </property>
  </widget>
  <widget class="QLabel" name="keyLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Key for Random</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="stegoTypeGroupBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Steganography Encode Type</string>
   </property>
   <widget class="QRadioButton" name="lsbRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>LSB</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="msbRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>MSB</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="randomBitRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Random bit</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>330</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="musicBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Music file</string>
   </property>
  </widget>
  <widget class="QLabel" name="doneLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="aesKeyLineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1234567890qazwsxedcrfvtgbyhnujmi</string>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
  </widget>
  <widget class="QLabel" name="aesKeyLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>AES256 Key</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
